examples/with-solidstart/README.md

```
@@ -0,0 +1,63 @@
+   ```jsx
+   import { clientOnly } from "@solidjs/start/client";
+
+   export default clientOnly(async () => ({ default: MyPage }));
+   ```
+
+For more details, refer to the `clientOnly` [documentation](https://docs.solidjs.com/solid-start/reference/client/client-only#clientonly).
+
+<div align="center">
+   </br>
+   </br>
+  <img src="public/logo.svg" width="300px">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)